src/pipecat/transports/base_output.py

```diff
@@ -386,10 +386,13 @@ async def _cancel_camera_task(self):
     async def _draw_image(self, frame: OutputImageRawFrame):
         desired_size = (self._params.camera_out_width, self._params.camera_out_height)
 
+        # TODO: we should refactor in the future to support dynamic resolutions
+        # which is kind of what happens in P2P connections.
+        # We need to add support for that inside the DailyTransport
```

Comment:
What do you think, is it something we wish to support ?

Reply:
I think there are two back to back webrtc, so the quality or resolution would be determined by the link that is constrained.

Reply:
What I mean is that currently, inside Pipecat, the DailyTransport always works with a fixed resolution and resizes the input if it doesn’t arrive in the expected resolution.